src/models.ts

```diff
@@ -96,6 +97,7 @@ export interface IReportCreateConfiguration {
    accessToken: string;
    datasetId: string;
```

Comment:
strange ... datasetId is part of the createReport configuration and not of loadReport configuration ...